website/app/styles/typography/index.scss

```diff
@@ -1,2 +1,56 @@
 @import "./font-faces";
 @import "./mixins";
+
+// TYPOGRAPHIC STYLES
+
+// HEADERS
+// TODO! discuss with Heather if we use `docs-text-h1` or `docs-text-display1`
```

Comment:
For me, I would prefer to reference the tag names because I think that makes the class names clearer and easier to use and it also matches what's in the Figma doc: https://www.figma.com/file/Ky0qWjvHZR3je1lCBlzNB1/HDS-website?node-id=181%3A23422 

(I don't know how formal that Figma is though or if it's just a rough representation of curent visual styles.)